### ISSUE TITLE
PHP 8.1.11 errors

### ISSUE THREAD

**Issue by mariantanase** [conseilgouz/AutoReadMore-J4#2]
Hi,
When I switch to PHP 8.1.11, error messages are shown on the home page :

```
Deprecated: explode(): Passing null to parameter #2 ($string) of type string is deprecated in C:\bearsampp\www\demosite\plugins\content\autoreadmore\autoreadmore.php on line 540
Deprecated: explode(): Passing null to parameter #2 ($string) of type string is deprecated in C:\bearsampp\www\demosite\plugins\content\autoreadmore\autoreadmore.php on line 548
```

Can be fixed ?

Thanks

**Comment by mariantanase** [comment]
I solved it this way, I hope it's correct.

Edit **autoreadmore.php** and replaced:

`$item_ids = array_map('trim', explode(',', $item_ids)); `

with

`$item_ids = array_map('trim', explode(',', $item_ids ?? ''));`

also replaced:

`$category_ids = array_map('trim', explode(',', $category_ids));`

with:

$category_ids = array_map('trim', explode(',', $category_ids ?? ''));

**Comment by conseilgouz** [comment]
Thank you for reporting this issue.

Your fix is included in version 5.0.7.

Regards,

Pascal
